docs/content/ja/docs/user-guide/triggering-a-deployment.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Deployment のトリガー"
+linkTitle: "Deployment のトリガー"
+weight: 2
+description: >
+  Deployment の自動トリガーや手動でのトリガーについて
+---
+
+PipeCD は Git をシングルソースオブトゥルース（全ての Application リソースが Git で宣言的かつ不変で定義されている）として扱っています。
```

Comment:
In a Japanese post, it tends to be this.
https://ja.wikipedia.org/wiki/%E4%BF%A1%E9%A0%BC%E3%81%A7%E3%81%8D%E3%82%8B%E5%94%AF%E4%B8%80%E3%81%AE%E6%83%85%E5%A0%B1%E6%BA%90

```suggestion
PipeCD は Git を信頼できる唯一の情報源（全ての Application リソースが Git で宣言的かつ不変で定義されている）として扱っています。
```